doc/Installation/Installation-(RHEL-CentOS).md

```diff
@@ -193,23 +202,41 @@ To prepare the web interface (and adding devices shortly), you'll need to create
 
 First, create and chown the `rrd` directory and create the `logs` directory
 
+```bash
     mkdir rrd logs
     chown -R librenms:librenms /opt/librenms
+    chmod 775 rrd
+
     # For HTTPd (Apache):
     chown apache:apache logs
+
     # For Nginx:
     chown nginx:nginx logs
-
-    chmod 775 rrd
+```
 
 > If you're planing on running rrdcached, make sure that the path is also chmod'ed to 775 and chown'ed to librenms:librenms.
 
+**SELinux**
+> if you're using SELinux you need to allow nginx to write into logs directory.
+> semanage tool is a part of policycoreutils-python, so if don't have it, you can install it
+
```

Comment:
Please note here that SELinux is still experimental to us, we cant guarantee it working yet